### PR TITLE
Full api versioning

### PR DESCRIPTION
**Pull request by erwagasore**
- Changed `get_url` to use `request` as an argument instead of `version`
- Removed the `create` routes in favor of `register`
- Added template for email confirmation
- Updated tests for users blueprint
- Fully coverage for `users` and maintained global coverage to `96%`

### CODE REVIEW

**Comment by erwagasore** [comment]
@kenessajr please review

File: api/app/awaremodel.py [hunk 1]
```diff
@@ -263,17 +266,25 @@ def remove(self):
         db.session.delete(self)
         db.session.commit()
 
-    def get_url(self, version=None):
-        return get_url(self, api_version=version)
+    def get_url(self, request=None):
+        return get_url(self, request=request)
```
Comment (kenessajr) [inline api/app/awaremodel.py]: @erwagasore you changed `api_versioning` to `request`.

File: api/app/templates/confirm.html [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
...firm your email address</div></td></tr><tr><td class="sc-htoDjs eOYkeo"><a href="{{confirm_url}}" color="blue" class="sc-bdVaJa sc-ifAKCX jgvSlL"><button font-size="12" font-weight="bold" color="white" class="sc-bdVaJa sc-EHOje NEXOO">Confirm your email</button></a></td></tr><tr><td class="sc-htoDjs OFcdl"><div class="sc-bdVaJa sc-htpNat bdCvlt">If you are having trouble with clicking the button, follow the link below instead, or copy and paste the link into your web browser</div></td></tr><tr><td class="sc-htoDjs hBngQS"><div color="#FB0320" font-size="12px" class="sc-bdVaJa sc-htpNat lmHfMh">{{confirm_url}}</div></td></tr><tr><td class="sc-htoDjs iIoFOg"><div class="sc-bdVaJa sc-htpNat bdCvlt">Best,<br>Team Pindo</div></td></tr></tbody></table><table class="sc-iwsKbI jGoobK"><tbody><tr><td class="sc-htoDjs IEKUO"><div class="sc-bdVaJa sc-htpNat bCjqUz">Pindo is a communication platform for humans &amp; machines</div></td></tr></tbody></table><table class="sc-iwsKbI eEfNhf"><tbody><tr><td class="sc-htoDjs bZzhDu"><img height="32" src="https://pindo.ams3.cdn.digitaloceanspaces.com/with_love_badge.png" class="sc-bdVaJa sc-bZQynM hNnhDk"></td></tr></tbody></table></div></body></html>
```
Comment (kenessajr) [inline api/app/templates/confirm.html]: you chose ams3 ('https://pindo.ams3.cdn.digitaloceanspaces.com') for space, is there any in lon1

Comment (erwagasore) [reply api/app/templates/confirm.html]: No there is no `London` that is why I did use `Amsterdam`

**Pull request merged**